app/assets/javascripts/modules/logger.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  function Logger() {
+    if (isDevelopmentEnvironment()) {
+      return console;
+    } else {
+      return silentConsole();
+    }
+  }
+
+  function isDevelopmentEnvironment() {
+    return /\.dev\.gov\.uk$/.test(window.location.hostname);
+  }
+
+  function silentConsole() {
+    var fakeConsole = {};
+
+    $.each(console, function(key) {
+      if (typeof console[key] === 'function') {
+        fakeConsole[key] = function() {};
+      }
+    });
+
+    return fakeConsole;
+  }
+
+  GOVUK.Logger = Logger;
+})();
```

Comment:
yep, i just see 56 mentions of it in code, just unsure as to when we may get all these bits deprecated. i just didnt know if it was causing problems, or jsut tryign to put people down a certain path